pydantic/_internal/_generate_schema.py

```diff
@@ -151,23 +151,23 @@ def generate_config(config: ConfigDict, cls: type[Any]) -> core_schema.CoreConfi
     """
     Create a pydantic-core config from a pydantic config.
     """
-    extra = None if config['extra'] is None else config['extra'].value
+    extra = None if config['extra'] is None else config['extra'].value  # type: ignore
     core_config = core_schema.CoreConfig(  # type: ignore[misc]
         **core_schema.dict_not_none(
-            title=config['title'] or cls.__name__,
+            title=config['title'] or cls.__name__,  # type: ignore
             extra_fields_behavior=extra,
-            allow_inf_nan=config['allow_inf_nan'],
-            populate_by_name=config['populate_by_name'],
-            str_strip_whitespace=config['str_strip_whitespace'],
-            str_to_lower=config['str_to_lower'],
-            str_to_upper=config['str_to_upper'],
-            strict=config['strict'],
-            ser_json_timedelta=config['ser_json_timedelta'],
-            ser_json_bytes=config['ser_json_bytes'],
-            from_attributes=config['from_attributes'],
-            loc_by_alias=config['loc_by_alias'],
-            revalidate_instances=config['revalidate_instances'],
-            validate_default=config['validate_default'],
+            allow_inf_nan=config['allow_inf_nan'],  # type: ignore
+            populate_by_name=config['populate_by_name'],  # type: ignore
+            str_strip_whitespace=config['str_strip_whitespace'],  # type: ignore
+            str_to_lower=config['str_to_lower'],  # type: ignore
+            str_to_upper=config['str_to_upper'],  # type: ignore
+            strict=config['strict'],  # type: ignore
+            ser_json_timedelta=config['ser_json_timedelta'],  # type: ignore
+            ser_json_bytes=config['ser_json_bytes'],  # type: ignore
+            from_attributes=config['from_attributes'],  # type: ignore
+            loc_by_alias=config['loc_by_alias'],  # type: ignore
+            revalidate_instances=config['revalidate_instances'],  # type: ignore
+            validate_default=config['validate_default'],  # type: ignore
```

Comment:
all these type ignores seem pretty unfortunate, especially if you can't say that it's due to possibly-missing value, as opposed to invalid key